modules/wordads/css/style.css

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
So in this case I would add to `frontendcss` list and remove the `wp_enqueue_style` from my module?